web_services/foxx/brightid/index.js

```diff
@@ -169,8 +174,22 @@ const handlers = {
       res.throw(403, 'user is not sponsored', {errorNum: NOT_SPONSORED});
     }
 
-    if (! db.userVerifications(user).includes(context.verification)) {
-      res.throw(404, 'user can not be verified for this context', {errorNum: CAN_NOT_BE_VERIFIED});
+    const verifications = db.userVerifications(user);
+    const allVerifications = ['BrightID', 'Yekta', 'DollarForEveryone', 'SeedConnected', 'SeedConnectedWithFriend']
```

Comment:
Can't we get the list of possible verifications from the db rather than hard-coding it?

Reply:
Iterating over all verifications in database on every call of this endpoint is too heavy.
@siftal removed this hard coded section in this way by getting variables of expression:
https://github.com/BrightID/BrightID-Node/commit/687e52ffa8b08ace210eea8a3c190e4cd0773dd6